tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot

```diff
@@ -33,6 +33,17 @@ Launch Container
     ${id}=  Get Line  ${output}  -1
     [Return]  ${id}
 
+Launch Container With Port Forwarding
+    [Arguments]  ${name}  ${port1}  ${port2}  ${network}=default
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create -it -p ${port1}:80 -p ${port2}:80 --name ${name} --net ${network} ${nginx}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} start ${name}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    Wait Until Keyword Succeeds  20x  5 seconds  Hit Nginx Endpoint  %{VCH-IP}  ${port1}
+    Wait Until Keyword Succeeds  20x  5 seconds  Hit Nginx Endpoint  %{VCH-IP}  ${port2}
```

Comment:
Two ports forwarding to the same container port is not something I've seen used and one that could easily be missed in implementation - nice to see it in a test.

Reply:
New comment - the `Wait Until ...` here will only succeed in this form if `--net=bridge`.
My main PR adds `docker port` support, e.g.
```
$ docker run -d -p 80:8080 -p 81:8080 --net=bridge --name=bridge_only tomcat:alpine
$ docker port bridge_only 8080
192.168.78.127:80
192.168.78.127:81
$ docker run -d -p 80:8080 --net=external --name=published_redirect tomcat:alpine
$ docker port published_redirect 8080
192.168.78.129:80
$
```